### PR TITLE
Printable drills and constructable thrusters

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR 
adds lathe recipes and techs for both the mining drill and the power drill (since it happened to have a recipe)
Also adds recipes for thruster and gyroscope circuitboards, making them constructable.
**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame
- [ ] This PR does not require an ingame showcase

image
image
image

**Changelog**

:cl:
- add: Mining drills and power drills can now be printed at the protolathe.
- add: Thrusters and Gyroscopes can be built and upgraded.

### CODE REVIEW

**Comment by S1ss3l** [comment]
Gyroscope make sense but aint it weird that the thruster is made with a cricuitboard ?

File: Content.Server/Shuttles/Systems/ThrusterSystem.cs [hunk 1]
```diff
@@ -58,6 +59,9 @@ public override void Initialize()
 
             SubscribeLocalEvent<ThrusterComponent, ExaminedEvent>(OnThrusterExamine);
 
+            SubscribeLocalEvent<ThrusterComponent, RefreshPartsEvent>(OnRefrehParts);
```
Comment (deltanedas) [inline Content.Server/Shuttles/Systems/ThrusterSystem.cs:62]: typo OnRefrehParts

File: Content.Server/Shuttles/Systems/ThrusterSystem.cs [hunk 2]
```diff
@@ -515,6 +519,18 @@ public void SetAngularThrust(ShuttleComponent component, bool on)
             }
         }
 
+        private void OnRefrehParts(EntityUid uid, ThrusterComponent component, RefreshPartsEvent args)
```
Comment (deltanedas) [inline Content.Server/Shuttles/Systems/ThrusterSystem.cs:522]: typo OnRefrehParts

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by deltanedas — APPROVED** [review]
please merge please merge please merge please merge

**Pull request merged**